upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -494,7 +494,22 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 				Version:           fi.String(version),
 				Selector:          networkingSelector,
 				Manifest:          fi.String(location),
-				KubernetesVersion: ">=1.6.0",
+				KubernetesVersion: ">=1.6.0 <1.8.0",
```

Comment:
>=1.8.0?

Reply:
what do you mean? is the selector wrong?